File: articles/status-quo-bias-institutional-inertia/go/status_quo_choice.go
package main

import "fmt"

func utilityStatusQuo(valueStatusQuo, statusQuoPremium float64) float64 {
	return valueStatusQuo + statusQuoPremium
}

func utilityAlternative(valueAlternative, switchCost, lossAversion, perceivedLoss float64) float64 {
	return valueAlternative - switchCost - lossAversion*perceivedLoss
}

func chooseAlternative(valueStatusQuo, valueAlternative, premium, switchCost, lossAversion, perceivedLoss float64) bool {
	return utilityAlternative(valueAlternative, switchCost, lossAversion, perceivedLoss) >= utilityStatusQuo(valueStatusQuo, premium)
}

func main() {
	fmt.Printf("Synthetic alternative adoption under status quo bias: %v\n", chooseAlternative(0.50, 0.68, 0.08, 0.05, 1.50, 0.04))
}

File: articles/status-quo-bias-institutional-inertia/cpp/status_quo_choice.cpp
#include <iostream>

double utility_status_quo(double value_status_quo, double status_quo_premium) {
    return value_status_quo + status_quo_premium;
}

double utility_alternative(double value_alternative, double switch_cost, double loss_aversion, double perceived_loss) {
    return value_alternative - switch_cost - loss_aversion * perceived_loss;
}

bool choose_alternative(double value_status_quo, double value_alternative, double premium, double switch_cost, double loss_aversion, double perceived_loss) {
    return utility_alternative(value_alternative, switch_cost, loss_aversion, perceived_loss)
        >= utility_status_quo(value_status_quo, premium);
}

int main() {
    std::cout << "Synthetic alternative adoption under status quo bias: "
              << choose_alternative(0.50, 0.68, 0.08, 0.05, 1.50, 0.04)
              << std::endl;
    return 0;
}
